crates/api_decl/src/namespaces/zkos.rs

```diff
@@ -0,0 +1,13 @@
+use jsonrpsee::core::RpcResult;
+use jsonrpsee::proc_macros::rpc;
+
+/// API bindings for the `zkos` experimental namespace.
+#[rpc(server, namespace = "zkos")]
+pub trait ZKOSNamespace {
```

Comment:
IMHO this belongs to `anvil_zks` namespace along with other ZKsync-specific stuff (`commitBatch`, `proveBatch`, `executeBatch`). Or if we want a clear separation for Boojum maybe rename to `anvil_boojum`?

Reply:
I think `anvil_zks` is reasonable